src/bokeh/models/tools.py

```diff
@@ -524,6 +524,13 @@ def __init__(self, *args, **kwargs) -> None:
     A shaded annotation drawn to indicate the configured ranges.
     """)
 
+    select_gesture = Enum("pan", "tap", "none", default="none", help="""
```

Comment:
Is there another word we can use here besides "select"? Concerned about overloading this term and confusion with data selections. `start_gesture` maybe?

Reply:
I think this question boils down to what `RangeTool` actually is, because when I see this name, I ask `RangeWhat?Tool` and the answer is always `RangeSelectTool` (or `RangeSelectionTool`), thus everything in this PR (branch name, PR title, new properties, docstrings, etc.) involve the word `select`. I thought about alternatives, but nothing comes close in meaning.

> `start_gesture` maybe?

To me this is the same as above, `start_of_what?_gesture`. Maybe let's go generic and simply say `interaction_gesture`? Or perhaps we could be specific with `range_selection_gesture` and introduce "range selection" as a well defined term (in a glossary of terms) to make sure that we differentiate different types of selections. To me this is a situation similar to what we have with the term  "layout", which on its own is really context dependent and may mean server things: CSS/DOM layout, (our) computed layout (`LayoutDOM` and canvas layouts) and perhaps other.